prow/config/jobs/istio.yaml

```
@@ -217,6 +217,15 @@ jobs:
     type: presubmit
     command: [make, gen-check]
 
+  - name: checkReleaseNotes
+    type: presubmit
+    command:
+      - ./tools/check_release_notes.sh
+      - --token-path=/etc/github-token/auth
+    requirements: [github]
+    modifiers:
+      - skipped
```

Additionally including `optional` will prevent the status context (if run and failed) from blocking merge.
```suggestion
      - skipped
      - optional
```

---

Thanks.